llvm/lib/CodeGen/RegAllocFast.cpp

```diff
@@ -391,6 +391,8 @@ class RegAllocFastImpl {
   bool mayLiveOut(Register VirtReg);
   bool mayLiveIn(Register VirtReg);
 
+  bool isInlineAsmBrSpill(const MachineInstr &MI) const;
```

Comment:
Two things:
1. It needs a comment because at first "I was what is a inlineasm branch spill"
2. This is a misnomer because unless I'm mistaken the instruction may not be spilling a value coming from an inlineasm branch. I would go with `mayBeSpillFromInlineAsmBr`

Reply:
I had been pondering a few ones, and eventually I shortened the name too much. Updated, thanks!